Update get-git-repos

info-cppsp opened the pull request:
updated to python 3
replaced old repo names
replaced github link

info-cppsp commented:
@monsta @raveit65 @flexiondotorg @vkareh 
Please review.

((This lets you clone all mate repos at once to a directory.))

vkareh commented on get-git-repos/download_repos.py:
```diff
@@ -1,28 +1,57 @@
+        "python-caja"]
       
-GIT_SSH = "git clone git@github.com:mate-desktop/"
+GIT_SSH = "git clone https://github.com/mate-desktop/"
```
I use an SSH keypair for my repos, changing this to HTTPS would break things for me and drive me crazy (I don't even remember what my GitHub password is for that matter...). However, I haven't actually used this script - I just clone my repos manually

vkareh commented on get-git-repos/download_repos.py:
```diff
@@ -1,28 +1,57 @@
-GIT_SSH = "git clone git@github.com:mate-desktop/"
+GIT_SSH = "git clone https://github.com/mate-desktop/"
 
 # will prompt for the users ssh key on the first git clone 
```
nitpick: if you change from SSH to HTTPS, this comment is no longer relevant

info-cppsp commented:
I think this is for new users, not for people, who have their repos already set up.
I have many repos forked and I had to add the upstream / origin, so that I push to my forked repo.
Features can be added later.

I have a question:
This script doesn't have the execute bit enabled, but the others do.
Should I set it here as well?

updated: set execute bit, removed comment

vkareh commented:
Cool, thanks! Would be cool to have a flag set up for ssh: `./download_repos.py --ssh=true`, but based on your comments this looks fine

info-cppsp commented:
forced pushed new version, removed spaces on line ends, checked for tabs

info-cppsp commented:
@vkareh 
would you please test the new --ssh option?
I don't have permission or my github is just not set up properly...

based on the [argparse](https://docs.python.org/3.6/howto/argparse.html) tutorial.

vkareh commented:
Works great! Thanks :)
